digitalpy/core/persistence/value_change_event.py

```diff
@@ -1,5 +1,5 @@
-from digitalpy.core.event import Event
-from digitalpy.model.persistent_object import PersistentObject
+from digitalpy.core.util.event import Event
```

Comment:
**[pep8]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>D100 Missing docstring in public module